code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -391,6 +391,35 @@
 /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/missile_rack/flashbang/clusterbang/limited/rearm()
 	return//Extra bit of security
 
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/missile_rack/cleaner
```

Comment:
Please reorganize this type definition.
By convention, name should be first, followed by desc, icon_* if present, then vars at the very bottom.

The var should also be autodoc'd.

Also, you probably want a custom desc on this.